Automattic/wp-calypso
# Add search query parameter to JITM impression tracks events

Related to https://github.com/Automattic/dotcom-forge/issues/2681

## Proposed Changes

* Send the `search_query` parameter when tracking the `calypso_wpcom_spotlight_jitm_impression` event
## Testing Instructions



* Patch D113284-code and sandbox `public-api` to have test JITMs added on WPCOM
* Open the dev console and issue `localStorage.setItem( 'debug', 'calypso:analytics*' );` to see tracks events being fired
* Go to `/plugins/` and search for a plugin
* Filter the console for `calypso_wpcom_spotlight_jitm_impression` and check that the event props contain `search_query` with the correct value
* Check in tracks that the event is fired with the correct props
![CleanShot 2023-06-20 at 14 57 56@2x](https://github.com/Automattic/wp-calypso/assets/11555574/caf4b74b-fa19-4d1a-baf1-48f0a122bdc4)

## Pre-merge Checklist
